dotnet/VippsMobilePay.Changefeed/VippsMobilePay.Changefeed.sln.DotSettings

```diff
@@ -0,0 +1,3 @@
+﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
```

Comment:
Do we normally include this into git?

Reply:
We do not normally add it, but in this case I think it makes sense, as I am adding words to the dictionary.